youtube_dl/options.py

```diff
@@ -177,6 +177,11 @@ def _hide_login_info(opts):
         default=None, metavar='URL',
         help='Use the specified HTTP/HTTPS proxy. Pass in an empty string (--proxy "") for direct connection')
     network.add_option(
+        '--alternative-proxy', dest='alternative_proxy',
```

Comment:
There is something tricky. Take sohu.py (at yan12125/youtube-dl@f7b8b0c39e8fd6dfdddb85b5bed2812d3fd68d69) for example, not all traffic for info extraction need China IP. Only the one with hostname hot.vrs.sohu.com needs it. So the name extraction proxy may be misleading.